client/my-sites/email/email-management/home/email-list-inactive.jsx

```diff
@@ -7,8 +7,14 @@ import { emailManagementPurchaseNewEmailAccount } from 'calypso/my-sites/email/p
 
 class EmailListInactive extends React.Component {
 	render() {
-		const { selectedSiteSlug, currentRoute, domains, translate } = this.props;
-
+		const {
+			currentRoute,
+			domains,
+			header,
```

Comment:
Also, would this be better named as `headerComponent` seeing as it's actually a component prop ?